plantuml2mysql.py

```diff
@@ -71,23 +74,25 @@ def main():
             uml = False
             continue
         if not uml:
-             continue
+             continue     
         if l.startswith("class"):
             table = True; field = False
             primary = []; index = ""
             # Table names are quoted and lower cased to avoid conflict with a mySQL reserved word
-            print("CREATE TABLE IF NOT EXISTS `" + i[1].lower() + "` (")
+            print("CREATE TABLE stg." + i[1].lower() + " (")
             continue
         if table and not field and l == "==": # Seperator after table description
             field = True
             continue
         if field and l == "}":
             table = False; field = False
-            print("  PRIMARY KEY (%s)" % ", ".join(primary), end="")
+            if primary: #Vips
+                print("  PRIMARY KEY (%s)" % ", ".join(primary), end="")
             if index:
                 print(",\n%s" % index[:-2],)
                 index = ""
             print(");\n")
+            print("GO\n") #Vips
```

Comment:
I thought "go" doesn't work with mySQL as a delimiter by default. Is it really works with mysql without redefining it with `delimiter` command?

Reply:
Well frankly I changed this script to suite Azure SQL database. So may be if you find suitable you may create a separate branch for this. :)
And Of course #Vips  needs to be ignored. This was basically a TAG kind of thing which I added to mark the changes done by me.